Makefile

```diff
@@ -128,7 +131,11 @@ firmwares: stamp-clean-firmwares .stamp-firmwares
 	rm -rf $(IB_BUILD_DIR)
 	mkdir -p $(IB_BUILD_DIR)
 	$(eval TOOLCHAIN_PATH := $(shell printf "%s:" $(OPENWRT_DIR)/staging_dir/toolchain-*/bin))
+ifeq ($(BUILDTYPE),unstable)
```

Comment:
it might be, that some changes in the Makefile may have made this not required anymore. But initially I had some trouble here with multiple files matching.
Needs to be checked with current code